Index.html

```diff
@@ -1,89 +1,107 @@
-        integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg=="
-        crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <head>
+    <meta charset="utf-8" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+      integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <title>Lugares por conocer</title>
-</head>
+  </head>
 
-<body>
+  <body>
     <div class="cabecera">
-        <div class="logo">
-            <a href="#"><i class="bi bi-globe-central-south-asia"></i> GO</a>
-        </div>
-        <nav class="nav">
-            <a href="#">Contáctanos</a>
-        </nav>
+      <div class="logo">
+        <a href="#"><i class="bi bi-globe-central-south-asia"></i> GO</a>
+      </div>
+      <nav class="nav">
+        <a href="#">Contáctanos</a>
+      </nav>
     </div>
     <section>
-        <header class="content">
-            <h1 class="t1">LUGARES A VISITAR</h1>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum suscipit quo ipsam reiciendis dignissimos!
-                Facilis velit soluta consequatur est, aliquid fugit pariatur molestias qui et dignissimos doloremque
-                repellendus eaque doloribus.</p>
-            <div>
-                <a href="#lugares" class="btn">Descubrir</a>
+      <header class="content">
+        <h1 class="t1">LUGARES A VISITAR</h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Eum suscipit quo ipsam reiciendis
+          dignissimos! Facilis velit soluta consequatur est, aliquid fugit pariatur molestias qui et
+          dignissimos doloremque repellendus eaque doloribus.
+        </p>
+        <div>
+          <a href="#lugares" class="btn">Descubrir</a>
+        </div>
+      </header>
+      <div id="lugares">
+        <h2 class="t2">Sitios famosos para Visitar</h2>
+        <div class="cards">
+          <div class="box">
+            <img src="img/tulipanes.jpg" alt="" width="100%" height="50%" />
+            <h3 class="pt-2 px-1" style="margin: 10px">Jardín de Tulipanes</h3>
```

Comment:
```suggestion
            <h3 class="pt-2 px-1">Jardín de Tulipanes</h3>
```
@LizethDayannaSC no aplicar estilos en línea, todo desde el archivo css